src/main/java/com/xebialabs/overthere/smb/telnet/SmbTelnetConnection.java

```diff
@@ -49,8 +50,8 @@
      */
     public SmbTelnetConnection(String type, ConnectionOptions options, AddressPortMapper mapper) {
```

Comment:
I understood its a work in progress :) , wanted to commit the changes already made